src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable no-console */
+import React from 'react';
+import { uuid } from 'uuidv4';
+import users from '../../api/users';
+import '../../../node_modules/bootstrap/dist/css/bootstrap.min.css';
+
+type Props = {
+  todos: Todo[],
+  users: User[],
+  addTodo: (addTodo: Todo) => void,
+};
+
+type State = {
+  title: string;
+  name: string;
+  isTitleValid: boolean;
+  isNameValid: boolean;
+  isTitleTouch: boolean,
+  isNameTouch: boolean,
```

Comment:
no need to create these additional properties. You can just change isTitlValid and isNameValid in the needed places